src/systems/ec/implementations/vanilla/packages/ec-component-menu/ec-component-menu.scss

```diff
@@ -356,6 +356,19 @@
     }
 
     .ecl-menu:not(.ecl-menu--forced-mobile):not(.ecl-menu--forced-close) {
+      .ecl-menu__item--focused {
```

Comment:
I think that these rules can be merged with the `.ecl-menu__item:hover`ones below (to avoid duplication)

Reply:
@emeryro I tried to merge with .ecl-menu__item:hover before but Lint always complains about "no-descending-specificity" no matter what order I add them! Do you have any recommendation to bypass this?

Thanks!

Reply:
I added the following items to this PR:

- adding aria-hidden="true" on Icons
- adding <span className="ecl-u-sr-only"> (has submenu)</span> on Menu items with children
- make sure the menu can be closed using the esc key, both on mobile and desktop.